src/front/spv/mod.rs

```diff
@@ -694,10 +693,12 @@ impl<I: Iterator<Item = u32>> Parser<I> {
                     let base_id = self.next()?;
                     log::trace!("\t\t\tlooking up expr {:?}", base_id);
                     let mut acex = {
-                        let expr = self.lookup_expression.lookup(base_id)?;
+                        // the base type has to be a pointer, so we derefernce it here for the walthrough
```

Comment:
Nit: `walthrough` -> `walkthrough`? I think in technical terms: `traverse` or `traversal` would fit a little better.